docs/src/joint/api/dia/Link/intro.html

```diff
@@ -60,24 +57,23 @@ <h4 id="dia.Link.intro.presentation">Presentation</h4>
 &lt;g class=&quot;marker-arrowheads&quot;/&gt;
 &lt;g class=&quot;link-tools&quot; /&gt;</code></pre>
 
-    <p>As you can see, the link consists of a couple of SVG path elements and couple of SVG group elements:</p>
+<p>As you can see, the link consists of a couple of SVG path elements and couple of SVG group elements:</p>
 
-    <ul>
-        <li><code>.connection</code> is the actual line of the link.</li>
-        <li><code>.connection-wrap</code> is an SVG path element that covers the <code>.connection</code> element and is usually thicker so that the link is able to
-            handle pointer events (mousedown, mousemove, mouseup) that didn't target the thin <code>.connection</code> path exactly. This makes it easy
-            to &quot;grab&quot; the link even though the mouse cursor didn't point exactly at the (usually thin) <code>.connection</code> path element.</li>
-        <li><code>.marker-source</code> and <code>.marker-target</code> are the arrowheads of the link.</li>
-    </ul>
+<ul>
+    <li><code>.connection</code> is the actual line of the link.</li>
+    <li><code>.connection-wrap</code> is an SVG path element that covers the <code>.connection</code> element and is usually thicker so that the link is able to handle pointer events (mousedown, mousemove, mouseup) that didn't target the thin <code>.connection</code> path exactly. This makes it easy to <q>grab</q> the link even though the mouse cursor didn't point exactly at the (usually thin) <code>.connection</code> path element.</li>
+    <li><code>.marker-source</code> and <code>.marker-target</code> are the arrowheads of the link.</li>
+</ul>
 
-    <pre><code>link.attr({
-        '.connection': { stroke: 'blue' },
-        '.marker-source': { fill: 'red', d: 'M 10 0 L 0 5 L 10 10 z' },
-        '.marker-target': { fill: 'yellow', d: 'M 10 0 L 0 5 L 10 10 z' }
-    });</code></pre>
+<p>The subelements of the link can be optionally styled through link attributes. For example:</p>
 
+<pre><code data-lang="javascript">link.attr({
+    '.connection': { stroke: 'blue' },
+    '.marker-source': { fill: 'red', d: 'M 10 0 L 0 5 L 10 10 z' },
+    '.marker-target': { fill: 'yellow', d: 'M 10 0 L 0 5 L 10 10 z' }
+});</code></pre>
 
-<p>The <code>markup</code> attribute is not the only presentation attribute that may be specified for a <code>joint.dia.Link</code>. However, all the following Link properties are deprecated and included only for backwards compatibility. Use <a href="#linkTools">link tools</a> instead:</p>
+<p><div class="docs-important-note">The <code>markup</code> attribute is not the only presentation attribute that may be specified for a <code>joint.dia.Link</code>. However, the following four Link properties are deprecated and included only for backwards compatibility. Use <a href="#linkTools">link tools</a> instead.</div></p>
```

Comment:
I think `div` should be the parent of the `p` tag here. Think the `div` will close the `p` tag and render incorrectly in some cases.